### ISSUE TITLE
A11y> NavTabs> background-color> 1 violations is appearing 

### ISSUE THREAD

**Issue by Uditsingh67** [cengage/react-magma#886]
**Describe the bug**
Navtabs>> "1 violations" is appearing for "background-color".

**To Reproduce**
**Steps to reproduce the behavior:**

1. Go to 'https://storybook-preview-dev--upbeat-sinoussi-f675aa.netlify.app/?path=/story/navtabs--background-color
2. Click on 'Accessibility" add on
3. "1 violations" is appearing

**Screenshot** 
http://somup.com/c3jF6iuSfR

Expected behavior
"0 violations" should appear for NavTabs> background-color.

**Comment by silvalaura** [comment]
Fixed indirectly: 
![Image](https://github.com/user-attachments/assets/9876e3e9-b188-49eb-9749-d35ac89d28db)
